XamarinCommunityToolkitSample/Pages/Views/TabView/DynamicTabAddingPage.xaml

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<pages:BasePage
+    x:Class="Xamarin.CommunityToolkit.Sample.Pages.Views.TabView.DynamicTabAddingPage"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:pages="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages"
+    xmlns:views="http://xamarin.com/schemas/2020/toolkit">
+    <StackLayout>
+        <Button Clicked="OnAdd" Text="Add Three with Content Tab 3" />
```

Comment:
Could we modify the example a little bit to cover all the basic cases related with dynamic tabs? It would be interesting to have buttons for:

- Insert Tab
- Delete Tab
- Move Tab

Reply:
Sure. I can get to it hopefully soon. Was on holiday.

Reply:
Let us know if you can't make it happen this week, then we'll finish it. Want to get this in to release a new pre-release version :)

Reply:
Added test page options.

If you use them, you will see the tab view respond in some interesting ways. Inserts work on TabItems, but not TabItemsSource in the strip, or deleting makes them get in an odd appearing state.

Its certainly odd, but I am unsure if it has to do with my actual changes.

Feel free to take a look and see what changes we would want to make.